src/utils/events/eventListeners/eventListenerGenerator.ts

```diff
@@ -76,7 +90,7 @@ const getEventFilter = (
   eventSignature: string,
   contractAddress: string,
   clientType: ClientType,
-) => {
+): { topics: Array<string | null>; address?: string } => {
```

Comment:
I added a type for it in #40. Might be worth adding a TODO comment to update it here